docs/src/pages/[platform]/connected-components/authenticator/customization/AuthStyle.tsx

```diff
@@ -3,50 +3,43 @@ import {
   ThemeProvider,
   Theme,
   useTheme,
+  View,
 } from '@aws-amplify/ui-react';
 export function AuthStyle() {
   const { tokens } = useTheme();
   const theme: Theme = {
     name: 'Auth Example Theme',
     tokens: {
-      colors: {
-        background: {
-          primary: {
-            value: tokens.colors.neutral['90'].value,
+      components: {
+        authenticator: {
+          router: {
+            boxShadow: '0 0 16px hsla(0, 0%, 0%, 0.1)',
+            borderWidth: '0',
```

Comment:
Do the `tokens` no longer work for creating a `theme`? Also since `tokens` is used any longer `useTheme` is extraneous here i think?

Reply:
Oh good point, let me clean that up. They still work; I don't know if we have a strong decision tree for when to use `tokens.colors.neutral['90'].value` vs `'{colors.neutral.90}'` vs `var(--amplify-colors-neutral-90)` besides just vibes. @dbanksdesign?

Reply:
✨ vibes ✨  I would say, in general, `tokens.colors.neutral['90']` would be preferable because it has intellisense.